src/api/cvc4cpp.cpp

```diff
@@ -1280,6 +1280,143 @@ Sort OpTerm::getSort() const
 
 bool OpTerm::isNull() const { return d_expr->isNull(); }
 
+template <>
+std::string OpTerm::getIndices() const
+{
+  CVC4_API_CHECK_NOT_NULL;
+  std::string i;
+  Kind k = intToExtKind(d_expr->getKind());
+
+  if (k == DIVISIBLE_OP)
```

Comment:
Why is `DIVISIBLE_OP` here? It's index arg is uint32_t not string: 
https://github.com/CVC4/CVC4/blob/master/src/api/cvc4cpp.cpp#L2735
https://github.com/CVC4/CVC4/blob/master/src/api/cvc4cpp.cpp#L2749

Reply:
That’s actually incorrect, it’s an arbitrary precision integer: https://github.com/CVC4/CVC4/blob/57524fd9f204f8e85e5e37af1444a6f76d809aee/src/util/divisible.h#L34
I return it as a string to support arbitrary precision values.
`mkOpTerm` also supports building it with a string for the same reason: https://github.com/CVC4/CVC4/blob/57524fd9f204f8e85e5e37af1444a6f76d809aee/src/api/cvc4cpp.h#L1829

Reply:
Hm, your link to cvc4cpp.h actually confirms my above, currently we only have support for uin32_t. We should probably support both (it is quite inconvenient if you have to convert strings that fit into uint back and forth). String as argument should then also be supported in the API though, which it currently isn't. @mpreiner @4tXJ7f what do you think?

Reply:
Oh duh, I was reading the function declaration above the comment instead of below it, oops.

It is inconvenient to convert it, but I still don't really like the extra infrastructure needed for bounds checking... Either way, string as argument should be supported in the API right? Otherwise functionality is limited.

Reply:
We can do string and uint23_t (or change this to uint64_t actually) to create, and string to return (as you have now). Sounds reasonable?

Reply:
Yeah, sounds good to me!

Reply:
Sounds good.

Reply:
Added support for creating DIVISIBLE_OP with a string in f87e6bb4c5d7e4de31099973244b3b5fc4874316